cmd/hostpaths/hostpaths.go

```diff
@@ -420,6 +433,32 @@ func mapHostPaths(ctx context.Context, pManager, vManager manager.Manager) {
 	}, time.Second*5)
 }
 
+func filterManagedPods(ctx context.Context, options *context2.VirtualClusterOptions, podList *corev1.PodList, pManager manager.Manager) error {
+	if options.MultiNamespaceMode {
+		// Limit Pods
+		nsList := &corev1.NamespaceList{}
+		err := pManager.GetClient().List(ctx, nsList, &client.ListOptions{
+			LabelSelector: labels.SelectorFromSet(labels.Set{
+				namespaces.VclusterNamespaceAnnotation: options.TargetNamespace,
```

Comment:
I'd maybe go the other way around, 
```go
if !options.MultiNamespaceMode {
    return nil
}
...
```
so we don't have the whole code inside a if block

Reply:
changed